dataprofiler/profilers/utils.py

```diff
@@ -59,19 +60,31 @@ def shuffle_in_chunks(data_length, chunk_size):
     :return: list of shuffled indices of chunk size
     """
     indices = KeyDict()
+    rng = np.random.default_rng()
     j = 0
+    
     # loop through all chunks
     for chunk_ind in range(max(math.ceil(data_length / chunk_size), 1)):
 
         # determine the chunk size and preallocate an array
         true_chunk_size = min(chunk_size, data_length - chunk_size * chunk_ind)
         values = [-1] * true_chunk_size
 
+        
+        # Generate random list of indicies         
+        list_j = j
+        lower_bound_list = np.zeros([true_chunk_size])
+        for i in range(true_chunk_size):
+            lower_bound_list[i] = list_j
+            list_j+=1
```

Comment:
can it just be `lower_bound_list = np.array(range(j, j + true_chunk_size))`

Reply:
I believe you're correct.

Reply:
Current`` method:
```
10049030 function calls in 11.812 seconds

   Ordered by: internal time

   ncalls  tottime  percall  cumtime  percall filename:lineno(function)
     1001   10.818    0.011   11.693    0.012 shuffle_in_chunks.py:52(shuffle_in_chunks_v2)
 10000001    0.481    0.000    0.481    0.000 shuffle_in_chunks.py:14(__missing__)
     4000    0.225    0.000    0.225    0.000 {method 'items' of 'dict' objects}
     1000    0.136    0.000    0.389    0.000 {method 'integers' of 'numpy.random._generator.Generator' objects}
        1    0.119    0.119   11.812   11.812 shuffle_in_chunks.py:178(test_chunk_v2)

```

Your proposal:
```
10049029 function calls in 11.453 seconds

   Ordered by: internal time

   ncalls  tottime  percall  cumtime  percall filename:lineno(function)
     1001    9.681    0.010   11.336    0.011 shuffle_in_chunks.py:52(shuffle_in_chunks_v2)
     1000    0.797    0.001    0.797    0.001 {built-in method numpy.array}
 10000000    0.488    0.000    0.488    0.000 shuffle_in_chunks.py:14(__missing__)
     4000    0.227    0.000    0.227    0.000 {method 'items' of 'dict' objects}
        1    0.116    0.116   11.453   11.453 shuffle_in_chunks.py:178(test_chunk_v2)
```